### PR TITLE
Openshift static hostname fix

### PR DESCRIPTION
**Pull request by psykulsk**
RedhatCoreos launched by the Openshift installer is configured with a dynamic hostname. Because we disable NetworkManager which manages the hostname, we have to set a static hostname.

### CODE REVIEW

File: deploy/openshift/sources/static_hostname.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+while true; do
+  FQHOSTNAME=$(hostname -f)
+  echo "Setting static hostname to $FQHOSTNAME"
+  hostnamectl set-hostname $FQHOSTNAME
+  if [[ $? -eq 0 ]]; then
+    break;
+  fi
+done
+
```
Comment (Dysproz) [inline deploy/openshift/sources/static_hostname.sh:19]: Remove this

**Review by Dysproz — CHANGES_REQUESTED** [review]

**Review by Dysproz — APPROVED** [review]

**Review by katrybacka — APPROVED** [review]

**Comment by psykulsk** [comment]
recheck ha

**Comment by psykulsk** [comment]
recheck ha

**Comment by psykulsk** [comment]
recheck ha

**Comment by psykulsk** [comment]
recheck ha

**Pull request merged**